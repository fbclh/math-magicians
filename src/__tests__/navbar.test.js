import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Navbar from '../components/Navbar';

it('Check if Navbar page renders correctly', () => {
  const tree = renderer.create(
    <BrowserRouter>
      <Navbar />
    </BrowserRouter>,
  );
  expect(tree).toMatchSnapshot();
});
