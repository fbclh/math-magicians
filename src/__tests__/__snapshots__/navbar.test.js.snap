// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if Navbar page renders correctly 1`] = `
<nav
  className="navbar"
>
  <h1
    className="header"
  >
    Math Magicians
  </h1>
  <ul
    className="ul-links"
  >
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        aria-current="page"
        className="links active"
        exact={true}
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        className="links"
        exact={true}
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li
      className="li-links"
    >
      <a
        activeClassName="active-link"
        className="links"
        exact={true}
        href="/quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </li>
  </ul>
</nav>
`;
